site/src/pages/release-notes/0.99.6.mdx

```diff
@@ -0,0 +1,117 @@
+## 🌟 New features
+
+- New <type://CircuitBreakerRule> and <type://CircuitBreakerRuleWithContent> API lets you define a complex
+  circuit breaking policy conveniently. #2726
+  ```java
+  CircuitBreakerRule myRule =
+      CircuitBreakerRule.of(
+          // A failure if an Exception is raised.
+          CircuitBreakerRule.onException(),
+          // Neither a success nor a failure because the request has not been handled by the server.
+          CircuitBreakerRule.builder()
+                            .onUnprocessed()
+                            .thenIgnore(),
+          // A failure if the response is 5xx.
+          CircuitBreakerRule.onServerErrorStatus(),
+          // A success if the response is 2xx.
+          CircuitBreakerRule.builder()
+                            .onStatusClass(HttpStatusClass.SUCCESS)
+                            .thenSuccess(),
+          // Neither a success nor a failure. Do not take this response into account.
+          CircuitBreakerRule.builder().thenIgnore());
+  ```
+- You can now easily bind an MDC property to a <type://RequestContext> and log it using a logging framework.
+  #2634 #2724 #2729
+  ```java
+  ServiceRequestContext sctx = ...;
+  try (SafeCloseable ignored = sctx.push()) {
+      RequestScopedMdc.put(sctx, "transactionId", "1234");
+
+      ClientRequestContext cctx = ...;
+      try (SafeCloseable ignored2 = cctx.push()) {
+          assert MDC.get("transactionId").equals("1234");
+      }
+  }
+  ```
+- You can now install a custom hook to <type://RequestContextStorage> in runtime, usually at startup time. #2723
+  ```java
+  RequestContextStorage.hook(delegate -> new RequestContextStorageWrapper(delegate) {
+      @Nullable
+      @Override
+      public <T extends RequestContext> T push(RequestContext toPush) {
+          MDC.put("foo", toPush.attr(FOO)); // Usually, you can just use RequestScopedMdc for this purpose.
```

Comment:
Move the comment up so that the line is not too long? That being said, please check the release note page code snippets do not overflow horizontally.